lms/product/canvas/_plugin/grouping.py

```diff
@@ -112,11 +112,6 @@ def sections_enabled(self, request, application_instance, course):
 
         return course.settings.get("canvas", "sections_enabled")
 
-    def get_group_set_id(self, request, _assignment, historical_assignment=None):
```

Comment:
This is no longer neccesary, the default implementation:

https://github.com/hypothesis/lms/pull/6113/files#diff-5b637926713d9757746c5577144952e246fae09060f817fddce867c1719fe0abR90

Returns the config's group_set_id. 

What's needed is overriding  get_assignment_configuration for canvas, that's done here:
https://github.com/hypothesis/lms/pull/6113/files#diff-14ff2997632559124e0837f25ec10d3eafa125b25107b581d53d9ee5b2da1db6R41